components/navigation/NavWithHamburger.tsx

```diff
@@ -0,0 +1,118 @@
+"use client";
```

Comment:
This is basically just a copy of the old Navigation component, before the edits made in this PR which remove the hamburger.